src/app/shared/change-password-dialog/change-password-dialog.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material';
+import { UsersService } from 'app/users/users.service';
+import { FormBuilder, Validators } from '@angular/forms';
+@Component({
+  selector: 'mifosx-change-password-dialog',
+  templateUrl: './change-password-dialog.component.html',
+  styleUrls: ['./change-password-dialog.component.scss']
+})
+export class ChangePasswordDialogComponent implements OnInit {
+
+  /**
+   * Change Password Form
+   */
+  changePasswordForm: any;
+  password: string;
+  repeat_password: string;
+
+  /**
+   * @param {MatDialogRef} dialogRef Component reference to dialog.
+   * @param {any} data Provides a deleteContext.
+   */
+  constructor(public dialogRef: MatDialogRef<ChangePasswordDialogComponent>,
```

Comment:
Please format this function properly. Take a look at other components as to how to format these.